test/general.test.js

```diff
@@ -186,6 +186,38 @@ describe('Read theme', function () {
             done();
         });
     });
+
+    it('Can extract custom templates', function (done) {
```

Comment:
@kevinansfield If you can think of more test cases (e.g. more example file names), let me know 👍

Reply:
Only thing I can think of testing would be nested templates, eg `post-partials/footer.hbs`.

I know the `(^[^\/]+).hbs$` regex protects against those but it seems like one of those things that could regress if we allow deeper nesting at some point.

Reply:
I can add a test to show that if you add a `post-partials` folder, it won't be counted as a custom template. Is this what you mean? 🙈

Reply:
Yeah, I was just thinking of adding a `post-partials/footer.hbs` template to the test theme. The length checks should already act as a test case for it not being included?

Reply:
Okay can do, will add as second commit.

Reply:
https://github.com/TryGhost/gscan/pull/87/commits/59d66b1a1b24a4071608c36b9db5dc8561e2f872